performance_regression_test.py

```diff
@@ -105,17 +137,116 @@ def display_results(self, results):
         test_xml = ""
         for single_result in results:
             self.display_single_result(single_result)
-            test_xml += self.get_test_xml(single_result)
+            test_xml += self.get_test_xml(single_result, test_name=test_name)
 
         f = open(os.path.join(self.logdir,
                               'jenkins_perf_PerfPublisher.xml'), 'w')
-        content = """<report name="simple_regression_test report" categ="none">
+        content = """<report name="%s report" categ="none">
 %s
-</report>""" % test_xml
+</report>""" % (test_name, test_xml)
 
         f.write(content)
         f.close()
 
+    def add_stress_cmd_params(self, result, cmd):
+        # parsing stress command and return dict with params
+        cmd = cmd.strip()
+        cmd = cmd.strip().split('cassandra-stress')[1].strip()
+        if cmd.split(' ')[0] in ['read', 'write', 'mixed']:
+            section = 'cassandra-stress ' + cmd.split(' ')[0]
+            result[section] = {}
+            if cmd.startswith('no-warmup'):
```

Comment:
The `cmd` string always starts with 'read', 'write' or 'mixed', you check it in if statement.
So you can't see 'no-warmup' variable in json file.

The second bug is that we can't assume the command order here, so we can replace 'startswith' by '__contains__'.